src/app/sign_in/sign_in.scss

```diff
@@ -0,0 +1,167 @@
+/* latin */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 300;
+  src: local('Open Sans Light'), local('OpenSans-Light'), url(https://fonts.gstatic.com/s/opensans/v13/DXI1ORHCpsQm3Vp6mXoaTRampu5_7CjHW5spxoeN3Vs.woff2) format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
+}
+/* latin */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Open Sans'), local('OpenSans'), url(https://fonts.gstatic.com/s/opensans/v13/cJZKeOuBrn4kERxqtaUH3ZBw1xU1rKptJj_0jans920.woff2) format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
+}
+/* latin */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Open Sans Bold'), local('OpenSans-Bold'), url(https://fonts.gstatic.com/s/opensans/v13/k3k702ZOKiLJc3WVjuplzBampu5_7CjHW5spxoeN3Vs.woff2) format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
+}
+
+@font-face {
+  font-family: "Open Sans Semibold";
+  src: url("/assets/fonts/OpenSans-Semibold.eot") format("eot"),
+       url("/assets/fonts/OpenSans-Semibold.woff") format("woff"),
+       url("/assets/fonts/OpenSans-Semibold.ttf") format("truetype"),
+       url("/assets/fonts/OpenSans-Semibold.svg#OpenSans-Semibold") format("svg");
+  font-weight: normal;
+  font-style: normal;
+}
+
+.friends {
+  background: white url("/assets/images/smilies@3x.png") top left no-repeat;
+  background-size: contain;
+  display: inline-block;
+  height: 7.8em;
+  margin: 7em auto 0.8em auto;
+  width: 13.3em;
+}
+
+.title {
+  color: #000000;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 2em;
+  font-weight: bold;
+  letter-spacing: 0.3em;
+  margin: 0.8em auto 0.65em auto;
+}
+
+.slogan {
+  color: #212121;
+  font-family: 'Open Sans Semibold';
+  font-size: 1.8em;
+  font-weight: 600;
+  line-height: 1.8;
+  margin-top: 0.65em;
+}
+
+.main-text {
+  color: #000000;
+  font-family: 'Open Sans';
+  font-size: 14px;
+  line-height: 1.9;
+  margin: 0 auto 2.5rem auto;
+  max-width: 321px;
+  min-width: 300px;
+}
+
+.form-label {
+  color: #000000;
```

Comment:
Use `black` instead of the hexa value. When you have the chance use the defined colors instead of their hexa value.